.github/workflows/test.yml

```diff
@@ -94,3 +96,31 @@ jobs:
       - name: Test with pytest
         run: |
           python -m pytest
+
+# We might want to consider moving this to a new workflow if we add addition test jobs
+  notifications:
+    name: Notifications
+    runs-on: ubuntu-22.04
+    needs: [test]
+    if: always()
```

Comment:
[Examples](https://docs.github.com/en/actions/learn-github-actions/expressions#always) put this in `${{ }}`. Is that not needed?

Reply:
Maybe it's only needed at the step level.